src/main/java/com/google/sps/servlets/SessionInit.java

```diff
@@ -1,7 +1,47 @@
 package com.google.sps.servlets;
 
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.SessionCookieOptions;
+import com.google.sps.firebase.FirebaseAppManager;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
 import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 @WebServlet("/init")
-public class SessionInit extends HttpServlet {}
+public class SessionInit extends HttpServlet {
+  // Return empty string if no comment, otherwise return text
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
This should be moved to the `RequestUtils` class. It can be used across multiple servlets.